crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -300,6 +308,7 @@ impl ColumnType {
     /// Returns the precision of a ColumnType if it is converted to a decimal wrapped in Some(). If it can not be converted to a decimal, return None.
     pub fn precision_value(&self) -> Option<u8> {
         match self {
+            Self::TinyInt => Some(1_u8),
```

Comment:
hi @Dustin-Ray, what should be precision_value for TinyInt?

Reply:
hey good, question, so the precision value here represents the number of digits that this type can accommodate. ```TinyInt``` represents a signed byte type in the range of -128 to 127, so the maximum number of digits it can store is 3, which is what we should set for ```TinyInt``` in this case. Hope this helps!